components/Buttons/src/MDCButton.m

```diff
@@ -727,11 +724,7 @@ - (void)setBorderColor:(UIColor *)borderColor forState:(UIControlState)state {
 #pragma mark - Border Width
 
 - (CGFloat)borderWidthForState:(UIControlState)state {
-  NSNumber *borderWidth = _borderWidths[@(state)];
-  if (borderWidth) {
-    return (CGFloat)borderWidth.doubleValue;
-  }
-  return 0;
+  return (CGFloat)_borderWidths[@(state)].doubleValue;
```

Comment:
return (CGFloat)_borderWidth[@(state)].doubleValue doesn't convey to future engineers that undefined values are explicitly intended to be 0.

Please use the Elvis operator:
return _borderWidth[@(state)] ? (CGFloat)_borderWidths[@(state)].doubleValue : (CGFloat)0;

Reply:
Done. However, I added a `nil` check, since that is exactly what the analyzer complains about.